schema.json

```diff
@@ -845,9 +845,8 @@
                       "with": {
                         "oneOf": [
                           {
-                            "type": "array",
+                            "type": "string",
                             "description": "List of existing or new elements for single-dimension Build Matrix",
```

Comment:
FWIW This kind of smells like a bug in the server-side pipeline validator (that it forces string and doesn't allow int or bool). I tried making the `with:` value an int and it _did_ error, so this new schema is accurate.